server/paymentProviders/transferwise/index.ts

```diff
@@ -96,7 +96,7 @@ async function quoteExpense(
   const isExistingQuoteValid =
     expense.feesPayer !== 'PAYEE' &&
     expense.data?.quote &&
-    expense.data.quote['paymentOption'] &&
+    expense.data.quote['paymentOption']?.disabled === false && // We can not reuse quotes if there's no payment option available
```

Comment:
The code can look incorrect for people not familiar with the data structure. I know it's the same, but the following would look a bit more explicit, maybe with attached comments on each line.

```
// We want a paymentOption to be there
expense.data.quote['paymentOption'] &&
// It needs to be explicitely not disabled
expense.data.quote['paymentOption'].disabled === false
```